server/src/main/java/io/druid/curator/inventory/CuratorInventoryManager.java

```diff
@@ -69,6 +70,8 @@
 
   private volatile PathChildrenCache childrenCache;
 
+  private final CountDownLatch latch = new CountDownLatch(1);
```

Comment:
can we call this `initializationDoneLatch` or something like that ? also , I think `doneInitializing` boolean variable can be removed and `latch.await(1 ms)` can be used to do the boolean check wherever necessary.